cypress.config.js

```diff
@@ -0,0 +1,8 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'https://engage-sphere.vercel.app/',
+  },
+  fixturesFolder: false
+})
```

Comment:
Procure deixar sempre adicionar uma nova linha ao final de um arquivo, para evitar problemas ou notificações geradas pelos sistemas de controle de versões como o Git.

Reply:
Creio que aqui já há uma linha em branco no final visto que o ícone de `No newline at end of file` não é exibido.